### PR TITLE
update dependencies

### PR DESCRIPTION
**Pull request by tacolopo**

### CODE REVIEW

**Comment by uint** [comment]
The `cosmwasm-*` deps were updated in #128.

If you'd like, feel free to rebase this and include only the other updates. I'll merge then!

File: Cargo.toml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 name = "{{project-name}}"
 version = "0.1.0"
 authors = ["{{authors}}"]
-edition = "2018"
+edition = "2021"
```
Comment (webmaster128) [inline Cargo.toml:5]: This diff is a very good one for new projects

**Comment by tacolopo** [comment]
> #128

Hi, maybe I'm confused but I thought 128 was for the main branch? "[webmaster128](https://github.com/webmaster128) merged 6 commits into [CosmWasm:main](https://github.com/CosmWasm/cw-template)" whereas mine is for the minimal

**Comment by tacolopo** [comment]
ok well then do with this pr what you want haha

**Comment by uint** [comment]
Ah, my bad. Thought it was for `main`. Happy to merge this, but deprecate the minimal branch later once we have "minimal through template logic".

**Comment by tacolopo** [comment]
makes sense to me, thanks!

**Review by uint — APPROVED** [review]

**Pull request merged**